home-switch-render

koljuchka93 opened the pull request:
add refreshPage to home btn and logo

koljuchka93 committed:
add refreshPage to home btn and logo

heorhii-aksiuk committed:
Merge branch 'main' into home-switch-render

heorhii-aksiuk commented:
Был конфликт

heorhii-aksiuk commented on src/partials/header.html:
```diff
@@ -10,7 +10,7 @@
       <ul class="menu list">
         <li class="menu__item active-page">
-            <button class="menu__button home-button" type="button">Home</button>
+            <button class="menu__button home-button" type="button" onClick="refreshPage()">Home</button></a>
```
Что делает JS в HTML? Нас не так учили. Создаём рефы и слушаем кнопку в JS файлах

koljuchka93 replied:
Удалила лишнее.

koljuchka93 committed:
delete onClick

koljuchka93 committed:
Merge branch 'home-switch-render' of https://github.com/heorhii-aksiuk/team-proj-js into home-switch-render

tuzegor commented:
Катя, обнови мейн. Там есть файл home-library-page-switch.js
там есть слушатели событий хоум и лайбрари, впиши туда свою функцию рендора для хоум

koljuchka93 committed:
Merge branch 'main' into home-switch-render

koljuchka93 committed:
home-switch-render

- moved my part of the code to a file "home-library-page-switch"
- fix from colors to variables in the button.scss

koljuchka93 committed:
home-switch-render

fix bug: missing <a href="">

heorhii-aksiuk commented:
![image](https://user-images.githubusercontent.com/81520242/141339855-d8ae55e4-12a4-4cd7-8128-014d08f44164.png)

heorhii-aksiuk reviewed (changes requested):
Надо пофиксить, чтобы не было как на картинке что я скинул выше
